README.md

```diff
@@ -65,13 +48,141 @@ In the image below you can find some score examples.
 
 More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
 
-![Ten Pin Score Example](images/example_ten_pin_scoring.png)
+<!-- ![Ten Pin Score Example](images/example_ten_pin_scoring.png) -->
 
-## Code Review
 
-In code review we'll be hoping to see:
 
-* All tests passing
-* The code is elegant: every class has a clear responsibility, methods are short etc.
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Note that referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want.
+# User stories
```

Comment:
Very well done! I didnt even consider making user stories. This would have been a clever way to break things down!!